sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -40,6 +40,13 @@ static void _rpl_trickle_send_dio(void *args)
 {
     gnrc_rpl_dodag_t *dodag = (gnrc_rpl_dodag_t *) args;
     ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
+
+    /* a leaf node does not send DIOs periodically */
```

Comment:
Ah, I see (: This must be handled, you are right. But this inconsistency check is for data-plane traffic and we are still missing consistency checks for such packets. However, this does not use the trickle timer to send out the `DIO`, but does so reactively when such an inconsistency is detected.